cmd/podman/diff/diff.go

```diff
@@ -73,6 +73,9 @@ func ValidateContainerDiffArgs(cmd *cobra.Command, args []string) error {
 		return errors.New("--latest and containers cannot be used together")
 	}
 	if len(args) == 0 && !given {
+		if !registry.IsRemote() {
```

Comment:
You have it backwards: podman-remote does not have `--latest`. Easy fix: remove the `!`